aws-resource-exporter-config.yaml

```diff
@@ -0,0 +1,9 @@
+default:
+  regions: "us-east-1"
+  timeout: 10s
+rds: {}
+vpc:
+  regions: "all"
```

Comment:
i'd rather be explicit here instead of using `all`. the configuration will end up going through app-interface, and we will not need to update regions in this code to support another region.

the list of regions we have privatelink infra in is defined in app-interface already: https://gitlab.cee.redhat.com/service/app-interface/-/blob/243f9e4fac808d050c8d0bf66e4903db28192442/data/aws/osd-privatelink-prod/account.yml#L12-33

Reply:
this will also allow us to turn `regions` from a string to a list, which i find preferable.

Reply:
Not having the magic all option is fine for me.

@iamkirkbater What's your opinion of always specifying the regions explicitly - is that fine as well, or would you rather keep the `all` option that dynamically retrieves the regions?

Reply:
If app-interface gives us the option to not have to maintain that list of regions in the future, then let's nix the `all` string and go for an explicit list (especially if that makes this easier to finish).  My only worry would have been that every time we wanted to expand to another region that we would have to update a config, but if we get that "for free" from app-interface then let's go forward with the easiest path for us here.